requirements.txt

```diff
@@ -0,0 +1,8 @@
+opencv-python>=3.4.5.20
+pafy>=0.5.5
+pandas>=0.25.2
+PyYAML>=5.1.2
+torch>=1.2.0
+torchvision>=0.4.0
+tqdm>=4.20.0
+requests>=2.20.0
```

Comment:
Only put minimal requirements, most of these dependencies are not used in your code

Reply:
Done